### PR TITLE
[t2158] Changed placeholder text

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/da66d9cca4bf366/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/da66d9cca4bf366/output.txt

Total script time: 2.26 mins

Server is running on http://pr1048.butyr.org/templates/basic/
Run the Test Suite: http://pr1048.butyr.org/test/

File: css/buttons.less [hunk 1]
```diff
@@ -3,26 +3,40 @@
 */
 
 @_btnLineHeight: 14px;
+@_btnTotalHeight: 35px; // It turns out that the total height of our buttons is 50px.
```
Comment (cadecairos) [inline css/buttons.less]: 35 or 50? :P

Comment (k88hudson) [inline css/buttons.less]: hahaha

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/d164b48fd5797bb/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/d164b48fd5797bb/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/dfe90112a2b6a17/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/dfe90112a2b6a17/output.txt

Total script time: 3.00 mins

File: src/editor/media-editor.js [hunk 2]
```diff
@@ -45,69 +33,88 @@ define( [ "util/lang", "editor/editor", "util/uri", "text!layouts/media-editor.h
...
+          saveBtn.classList.remove( "butter-disabled" );
+        } else {
+          saveBtn.classList.add( "butter-disabled" );
+        }
+      }
 
-      wrapper = __URL_INPUT_INNER_WRAPPER.cloneNode( true );
-      wrapper.appendChild( urlInput );
+      function onEnter( e ) {
+        console.log( e );
```
Comment (cadecairos) [inline src/editor/media-editor.js]: remove plz!

**Pull request merged**